cmd/ip-masq-agent/ip-masq-agent.go

```diff
@@ -48,6 +48,7 @@ const (
 // config object
 type MasqConfig struct {
 	NonMasqueradeCIDRs []string `json:"nonMasqueradeCIDRs"`
+	MasqueradeCIDRs    []string `json:"masqueradeCIDRs"`
```

Comment:
The current implementation of this API is confusing:
- `NonMasqueradeCIDRs` may not take effect depends on what is given in `MasqueradeCIDRs`.
- `MasqueradeCIDRs` doesn't cover all the CIDRs that will be masqueraded, which is not intuitive.

A cleaner but less powerful way would be making these two mutually exclusive, which should be enough for your use case mentioned on https://github.com/kubernetes-incubator/ip-masq-agent/issues/9#issue-264753768? e.g.
``` golang
	// NonMasqueradeCIDRs give the CIDRs that should not be masqueraded.
	NonMasqueradeCIDRs []string `json:"nonMasqueradeCIDRs"`
	// MasqueradeCIDRs give the CIDRs that should be masqueraded.
	MasqueradeCIDRs    []string `json:"masqueradeCIDRs"`
```

Reply:
I'm currently using a branch with this functionality to add masquerade for all connections to a specific CIDR range. 

For my case that could work, but possibly this might be more powerful if we give the user the option to determine the order of rules and whether they're masquerading or non-masquerading.